libgadget/blackhole.c

```diff
@@ -514,6 +820,24 @@ blackhole_accretion_postprocess(int i, TreeWalk * tw)
     double dtime = get_dloga_for_bin(P[i].TimeBin) / All.cf.hubble;
 
     BHP(i).Mass += BHP(i).Mdot * dtime;
+
+    /*************************************************************************/
+    
+    if(blackhole_params.BH_DRAG > 0){
+        double fac = 0;
```

Comment:
Please add comments explaining where the physics for this comes from, like a link to an equation in a paper somewhere.